src/app/api-server/src/app.rs

```diff
@@ -441,6 +443,22 @@ pub async fn init_dependencies(
         }
     }
 
+    assert!(
+        config.dataset_env_vars.encryption_key.is_some(),
+        "Dataset env var encryption key is required"
+    );
```

Comment:
Since this option is now mandatory, please also add a check to the helm-chart layout.